File: andrew035/PEOO/lde_3/Q4.java
import java.util.Scanner;
public class Q4{
    public static void main(String[]args){
        Scanner s;
        s = new Scanner(System.in);
        double[] V1 = new double[4];
        double[] V2 = new double[4];
        double[] V3 = new double[4];

        for(int i = 0 ; i < 4 ; i++){
            System.out.print("Informe o número: ");
            V1[i] = s.nextDouble();
        }
        for(int i = 0 ; i < 4 ; i++){
            System.out.print("Informe o número: ");
            V2[i] = s.nextDouble();
        }
        for(int i = 0 ; i < 4 ; i++){
            V3[i] = V1[i] + V2[i];

            System.out.println(V3[i]);
        }
    }
}

File: jairo015/fluxo/Q4.py
n1 = int(input("Informe n1: "))
n2 = int(input("Informe n2: "))
n3 = int(input("Informe n2: ")) 

if n1 < n2:
^   if n1 < n3:
        print("O menor número é: " + str(n1))
    else:
        if n2 < n1:
            if n2 < n3:
                print("O menor número é: " + str(n2))
        else:
            if n3 < n1:
                if n3 < n2:
                    print("O menor número é: " + str(n3))
